handlers/handler_test.go

```diff
@@ -1,6 +1,7 @@
 package handlers
```

Comment:
I think we should rename these test files to `test_common.go` like how the sdk does. Just a cleaner naming format

Reply:
These issues should be a micro-PR that comes later. lets avoid omnibus PRs